index.js

```diff
@@ -18,70 +18,68 @@ app.engine("handlebars",handlebars.engine);
 app.set("view engine","handlebars");
 app.set('port', process.env.PORT || 4000);
 app.use(express.static(__dirname +'/public'));
+
 app.use( function( req, res, next){
   res.locals.showTests = app.get(' env') !== 'production' && req.query.test === '1';
   next();
  });
+
 app.use(require('body-parser').urlencoded({extended:true}));
 //app.use(require('cookie-parser')(credentials.cookieSecret));
 //app.use(require('express-session')({
  //resave:false,
  //saveUninitialized:false,
 // secret:credentials.cookieSecret
 //}));
+
 //app.get("/", function(req,res){
 //    res.render("home");
 //});
+
+function getMenu(req) {
+  var menu = [];
+  var isAdmin = true;
```

Comment:
The `isAdmin` information should come from the session. Why was this not modified yet? Also, there should be a difference between being logged in and logged out and that is also not implemented yet.

Reply:
I have got the sign in to work but I could not get the log in to work. I thought we created the isAdmin right there or do I need to create it somewhere else?